File: hot-framework/src/test/resources/be/solidx/hot/test/nio/http/py/http-client-post-xml.py
print("Starting...")

options = {
	'url': 'http://localhost:8080/test',
	'type': 'POST',
	'headers':{
		'Content-type':'application/xml'
	},
	'data':{
		'name':'Damien',
		'age':8
	}
}

def done(data, status, response):
    print data.getElementsByTagName("name")
    server.stop()

def fail(response, status, error):
    error.printStackTrace()
    server.stop()

client.buildRequest(options).done(done).fail(fail)

File: hot-framework/src/test/resources/be/solidx/hot/test/nio/http/js/http-client-post-xml.js
hprint("Starting...")

var options = {
	url: 'http://localhost:8080/test',
	type: 'POST',
	headers:{
		'Content-type':"application/xml"
	},
	data:{
		name:'Damien',
		age:8
	}
}

client.buildRequest(options).done(function(document) {
	//hprint(document.getDocumentElement())
	server.stop()
}).fail(function(response, status, error) {
	//error.printStackTrace()
	server.stop()
}) 